### ISSUE TITLE
Full screen icon not visible on iPhone

### ISSUE THREAD

**Issue by intermedion** [dimsemenov/PhotoSwipe#1259]
The lightbox full screen icon is not visible on iPhone Chrome or Safari browsers when using an actual iPhone device. The navigation arrows are also missing, even when there is space to the left and right of the lightbox image and the caption and the other icons are visible (numbering, links, and close icons).

Rotating the device from portrait to landscape orientation automatically goes in full screen mode. Tapping on the icon location, when in full screen mode, exits full screen and makes the browser top nav bar visible. 

The share and close buttons are visible, as well as the captions. 

The above can be verified on the PhotoSwipe site demo. They are also apparent on my own site, so it is not only the demo.

The icon and navigation arrows are visible for all available devices when the lightbox is opened in the Mac desktop Chrome Developer Tools/Toggle Device Toolbar.

iOS 10, macOS 10.12.2, Safari, Chrome - current versions

**Comment by linktoahref** [comment]
Hi,

I am also facing the same issue. 
The full screen icon is available on Android devices, however it is hidden for iPhone.

Tested on Android (Pixel 6A) and iPhone 11 against PhotoSwipe version 4.1.3
![android_gallery_screenshot](https://user-images.githubusercontent.com/15182823/191510864-6585eb00-3d3c-4164-bff6-2d144015562c.png)
![iphone_gallery_screenshot](https://user-images.githubusercontent.com/15182823/191510891-2589d40c-7429-4643-b5a6-9bf91901816d.png)

**Comment by acwolff** [comment]
> The lightbox full screen icon is not visible on iPhone Chrome or Safari browsers when using an actual iPhone device.

That is correct, because the full screen mode is not supported on an iPhone.

**Comment by linktoahref** [comment]
@acwolff Thanks for sharing that information. I wasn't aware about it. 😄
